mmseg/datasets/pascal_context.py

```diff
@@ -100,16 +100,15 @@ class PascalContextDataset59(BaseSegDataset):
                  [255, 71, 0], [0, 235, 255], [0, 173, 255], [31, 0, 255]])
 
     def __init__(self,
-                 ann_file: str,
+                 ann_file='',
                  img_suffix='.jpg',
                  seg_map_suffix='.png',
-                 reduce_zero_label=True,
+                 reduce_zero_label=False,
```

Comment:
And here it should be `True`.

Reply:
Thank you for your correction. I think I was confused because when I created the annotation maps, I had set the background to 255, and the category labels are 0-59, which caused me to set recude_zero_label to False in class ''PascalContextDataset59''. I've restored the settings for reduce_zero_label.